Lib/test/_test_multiprocessing.py

```diff
@@ -5849,6 +5850,24 @@ def test__all__(self):
         support.check__all__(self, multiprocessing, extra=multiprocessing.__all__,
                              not_exported=['SUBDEBUG', 'SUBWARNING'])
 
+    def test_spawn_sys_executable_none_allows_import(self):
+        # Regression test for a bug introduced in
+        # https://github.com/python/cpython/issues/90876 that caused an
+        # ImportError in multiprocessing when sys.executable was None.
+        # This can be true in embedded environments.
+        testfn = os_helper.TESTFN
+        self.addCleanup(os_helper.unlink, testfn)
+        with open(testfn, 'wb') as f:
+            f.write(b'''\
```

Comment:
The backslash isn't necessary, I think? Also, use the 'if 1:' trick used elsewhere in this test (in test_no_import_lock_contention) to avoid the wonky indentation.

Reply:
agreed, nicer. i also updated the test right above for consistency.